src/Apps/Collect/FilterState.tsx

```diff
@@ -36,9 +38,12 @@ const initialState = {
 
 export class FilterState extends Container<State> {
   state = cloneDeep(initialState)
+  tracking: any
```

Comment:
pretty sure we have an interface for `tracking` we can type this here

Reply:
I thought so too and looked through the project and so that it was pretty consistently typed as `any`.